ReviewNB-Demo.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Great to see you using lumpy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/guyawn/FoodInspections/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>